src/styles/index.scss

```diff
@@ -1,3 +1,116 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  color: #060b35;
+}
+
+.card {
+  $bg-color: #fff;
+
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  background-color: $bg-color;
+  border-radius: 5px;
+  padding-top: 33px;
+  padding-bottom: 17px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__info-block {
+    margin: 40px 17px 0;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 0 24px;
+  }
+
+  .stars {
+    display: flex;
+    align-items: center;
+  }
+
+  .stars__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+  }
+
+  .stars__star:last-child {
+    margin-right: 0;
+  }
+
+  .stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
It should works like in the task with stars, it means, that all modifiers from 0 to 5 should be available